ivadomed/main.py

```diff
@@ -4,24 +4,27 @@
 import shutil
 import sys
 import time
-
-import ivadomed.transforms as ivadomed_transforms
 import joblib
 import numpy as np
 import pandas as pd
+from tqdm import tqdm
+
 import torch
 import torch.backends.cudnn as cudnn
 import torch.nn as nn
-from ivadomed import loader as loader
+from torch import optim
+from torch.utils.data import DataLoader
+from torch.utils.tensorboard import SummaryWriter
+
 from ivadomed import losses
-from ivadomed import metrics
 from ivadomed import models
+from ivadomed import metrics
 from ivadomed import utils
+from ivadomed import loader as imed_loader
+from ivadomed import postprocessing as imed_postPro
```

Comment:
i would suggest to rename as imed_postpro (or imed_postproc) as the current convention in SCT and ivadomed is lowercase for variables/functions, all capital for global vars and camel for classes, but so far i haven't seen mixed case.